packages/app/src/templates/reducers/template.interface.ts

```diff
@@ -3,9 +3,26 @@
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
 
+import { PayloadAction } from "@reduxjs/toolkit"
+import { TemplatesEvent } from "App/templates/constants"
+import { CreateTemplateError } from "App/templates/errors"
 import { Template } from "App/templates/dto"
 
 export interface TemplateState {
   data: Template[]
```

Comment:
I'm thinking, maybe here `data` is too generic name? 
```suggestion
  templates: Template[]
```

Reply:
According to our style guild, the `data` key is used in each reducer. It makes reducers more generic and undrestandeble.